src/test/java/org/folio/rest/impl/SendNotifyTest.java

```diff
@@ -93,7 +95,9 @@ public static void setUp(TestContext context) {
     DeploymentOptions options = new DeploymentOptions().setConfig(new JsonObject().put(HTTP_PORT_JSON_PATH, port));
     vertx.deployVerticle(RestVerticle.class.getName(), options, result -> {
       try {
-        tenantClient.postTenant(null, x -> async.complete());
+        TenantAttributes attributes = new TenantAttributes()
+          .withModuleTo(String.format("mod-notify-%s", PomReader.INSTANCE.getVersion()));
```

Comment:
Why do we need these attributes?

Reply:
Its a RMB 29 change. Client that used an empty request must now include a body with at least: {"module_to":"module-id"}

Reply:
Thanks